lookbook/docs/patterns/10-page-headers.md.erb

```diff
@@ -0,0 +1,67 @@
+This component is an adaptation for OpenProject requirements of the React component `PageHeader` of Primer. This document will only state the points that differ from what **[Primer specifies](https://primer.style/components/page-header)** about this component in their documentation.
+
+## Anatomy
+
+There are minor adjustments on how the composition of this component is defined in Primer for OpenProject:
+
+**[Context bar](https://primer.style/components/page-header#context-bar):** Will be always present in all pages and will change depending on the viewport:
+
+- **Desktop:** The content of the context bar will be a [breadcrumb](https://primer.style/components/breadcrumbs).
+- **Tablet:** The content of the context bar will be a [breadcrumb](https://primer.style/components/breadcrumbs).
+- **Mobile:**
+  - The content of the breadcrumb will collapse to a single argument using the parent link style.
+  - All the page header actions will merge into an [ActionMenu](https://primer.style/components/action-menu) triggered by a single [Icon button](https://primer.style/components/icon-button) in the context bar.
+
+**[Title bar](https://primer.style/components/page-header#title-bar):** The default (medium) size will be always used keeping the capability as optionals of adding leading and trailing visuals and actions.
+
+**Actions:** A maximum amount of 5 actions can be displayed independently in the Page Header actions. If there are more than 5 actions the last one will be an [Icon button](https://primer.style/components/icon-button) with a more action, this one will trigger all the extra actions in a dropdown. All actions require their own icon.
+
+**Divider**: The divider bellow the Page Header will be always present with the exception of displaying a [Tav nav](https://primer.style/components/tab-nav) as navigation element bellow.
+
+## **Exceptional behaviours**
+
+- A special behaviour occurs when the page can be saved (or saved as) as one of the Page Header actions will be used to display an informational text and a link to perform the action.
+- Work packages details would be the only exceptional case where the breadcrumb will be used to display the parent work package instead of the navigational hierarchy.
```

Comment:
Suggested edit:

> - Work packages details are the only exceptional case, where the breadcrumb will be used to display the parent work package instead of the navigational hierarchy.